src/plot_erpimage.jl

```diff
@@ -57,7 +56,9 @@ function plot_erpimage!(
             sortindex = 1:size(plot, 2)
         else
             sortindex = sortperm(sortvalues)
-            ax.ylabel = "Trials sorted" #by $(mymacro(sortvalues))
+            if ax.ylabel[] == "Trials"
```

Comment:
if isnothing(sortindex) && !isnothing(sortvalues)
   config_kwargs!(config;axis=(;ylabel="Trials sorted")
end

Reply:
gosh, that was super easy